ansible_base/feature_flags/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import path, re_path
+
+from ansible_base.feature_flags import views
+from ansible_base.feature_flags.apps import FeatureFlagsConfig
+
+app_name = FeatureFlagsConfig.label
+
+api_version_urls = [
+    path('feature-flags/', views.FeatureFlagsListView.as_view(), name='featureflags-list'),
+    re_path(r'feature-flags/(?P<category_slug>[a-zA-Z0-9_]+)/$', views.FeatureFlagDetailView.as_view(), name='featureflags-detail'),
```

Comment:
Is this a detail for a specific single FLAG or a category of flags?

Is there a requirement to filter by category? if so, can we have another endpoint for getting a specific flag?

Reply:
Hey Bruno,
This is for a specific flag, not a category of flags. Will rename this